var/spack/repos/builtin/packages/go-md2man/package.py

```diff
@@ -6,41 +6,12 @@
 from spack import *
 
 
-class GoMd2man(Package):
+class GoMd2man(GoPackage):
     """go-md2man converts markdown into roff (man pages)"""
 
     homepage = "https://github.com/cpuguy83/go-md2man"
     url      = "https://github.com/cpuguy83/go-md2man/archive/v1.0.10.tar.gz"
 
     version('1.0.10', sha256='76aa56849123b99b95fcea2b15502fd886dead9a5c35be7f78bdc2bad6be8d99')
 
-    depends_on('go')
-
-    resource(name='blackfriday',
-             url='https://github.com/russross/blackfriday/archive/v1.5.2.tar.gz',
-             sha256='626138a08abb8579474a555e9d45cb5260629a2c07e8834428620a650dc9f195',
-             placement='blackfriday',
-             destination=join_path('src', 'github.com', 'russross'))
```

Comment:
This resource is no longer needed?

Reply:
For the versions that we support, its vendor dir has always contained what it needs.  We just weren't using it.  See https://github.com/spack/spack/issues/13023#issuecomment-557663683 for my notes.